.github/workflows/tests.yml

```diff
@@ -169,3 +167,207 @@ jobs:
+
+          echo COMPOSER_ROOT_VERSION=$SYMFONY_VERSION.x-dev >> $GITHUB_ENV
+          echo SYMFONY_VERSION=$SYMFONY_VERSION >> $GITHUB_ENV
+
+      - name: Install PHPUnit
+        run: |
+          mkdir -p ./vendor/symfony
+          cd ./vendor/symfony
+          ln -s ../../src/Symfony/Bridge/PhpUnit phpunit-bridge
+          cd ../..
+          ./phpunit install
+          echo PHPUNIT=`pwd`/phpunit >> $GITHUB_ENV
+
+      - name: Run tests
+        run: |
+          _run_tests() {
+            ok=0
+            echo "::group::$1"
+
+            # Install dependencies
+            cd ./$1
+            composer update --no-progress --ansi --prefer-lowest --prefer-stable 2>&1  || ok=1
+
+            if [ $ok -ne 0 ]; then
+              echo "::error::$1 failed"
+              return $ok
+            fi
+
+            # Run the tests
+            $PHPUNIT --colors=always --exclude-group tty,benchmark,intl-data 2>&1 || ok=1
+            echo ::endgroup::
+
+            if [ $ok -ne 0 ]; then
+              echo "::error::$1 failed"
+            fi
+
+            # Make the tests always pass because we don't want the build to fail (yet).
+            return $ok
+          }
+          export -f _run_tests
+
+          find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Bridge/PhpUnit/*' -print0 | xargs -0 -n1 dirname | sort | parallel _run_tests
+
+  phpunit:
+    name: PHPUnit
+    runs-on: Ubuntu-20.04
+
+    env:
+      extensions: 'amqp, apcu, igbinary, intl, mbstring, memcached, mongodb, redis'
+
+    strategy:
+      matrix:
+        php: ['7.1', '7.2', '7.3', '7.4', '8.0']
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Configure extensions
+        if: "${{ matrix.php == '8.0' }}"
+        # TODO: memcached is excluded because of compatibility issues with PHP 8
+        run: echo "extensions='amqp, apcu, igbinary, intl, mbstring, :memcached, mongodb, redis'" >> $GITHUB_ENV
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "${{ matrix.php }}"
+          extensions: "${{ env.extensions }}"
+          tools: flex
+
+      - name: Configure composer
+        run: |
+          COMPOSER_HOME="$(composer config home)"
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+          echo "COMPOSER_ROOT_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*').x-dev" >> $GITHUB_ENV
+
+      - name: Install dependencies
+        run: |
+          echo "::group::composer update"
+          composer update --no-progress --ansi
+          echo "::endgroup::"
+          echo "::group::install phpunit"
+          ./phpunit install
+          echo "::endgroup::"
```

Comment:
Nitpicking: We are inconstant with when we use quotes. Before here we only use quotes when needed. 

I suggest we always use quotes for clarity.